### DISCUSSION TITLE
Digitaler Zähler mit unterschiedlich großen Ziffern

### DISCUSSION THREAD

**Discussion by Wasserfritz** [jomjol/AI-on-the-edge-device#2027] [Q&A]
Hallo zusammen,
Ich habe einen digitalen Wasserzähler, bei dem die Ziffern für die m³ größer sind als die Ziffern für die Nachkommastellen.
Wenn ich in der Eingabemaske die Digitalen ROI's die Größe festlege, dann sieht auch alles gut aus; unterschiedliche Größen können eingegeben werden. Nach dem Save und Restart sind alle digits wieder gleich groß und die Erkennung funktioniert nicht.
Hat jemand einen Lösungsansatz?

**Comment by TopGoalVl** [comment]
![grafik](https://user-images.githubusercontent.com/119757556/218811272-d905c5ff-39ce-4393-9f0e-b24757633274.png)

Hallo Wasserfritz,

mach mal folgendes:
   - alle drei Haken entfernen
   - die ROI für die dig-Zähler anpassen und exakt positionieren
   - wichtig ist, dass alle ROIs korrekt gesetzt / positioniert sind
   - die Einstellungen **NUR sichern, KEINEN REBOOT** aus diesem screen durchführen
   - die Sicherung mit ok bestätigen
   - aus dem System Menü --> Info aufrufen
   - aus dem System Menü --> reboot aufrufen und starten.
  
**Der Grund:**
Das System merkt sich den gewählten Stand der o. g, Haken **NICHT** und setzt, bei einem erneuten Aufruf des screens, alle Haken als default neu. **M. E ist das ein bug.**

Wenn du den reoobot aus dem Setupscreen (dig-ROI) startest, wird dieser screen, nach dem reboot, erneut aufgerufen und alles, was du eingestellt hast, war für die Katz. Wenn du den screen nicht mehr aufrufst, bleien die Einstellungen konstant erhalten (natürlich nur bis zum nächsten screen Aufruf). Klingt verwirrend, ist aber so. Habe auch einige Zeit gebraucht, um diesen workaround zu finden.

Dieses Verhalten gibt es seit Version 13-0.x! Ich habe es nicht mehr getestet, vermute aber, dass es in der aktuellen V14.0.3 immer noch so ist.

**Comment by Wasserfritz** [comment]
Danke an TopGoalVI
Läuft jetzt prima

**Reply by TopGoalVl** [reply]
Nachtrag:
Sollte man den DIGROI screen mal aufrufen, werden die Rahmen verändert. Wenn man nicht speichert, bleiben die ursprünglichen Einstellungen natürlich erhalten.

**Comment by Wasserfritz** [comment]
Habe ich schon bemerkt, Danke für den Hinweis

 
 

Gesendet: Mittwoch, 15. Februar 2023 um 07:15 Uhr
Von: "TopGoalVl" ***@***.***>
An: "jomjol/AI-on-the-edge-device" ***@***.***>
Cc: "Wasserfritz" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [jomjol/AI-on-the-edge-device] Digitaler Zähler mit unterschiedlich großen Ziffern (Discussion #2027)


 

Nachtrag:
Sollte man den DIGROI screen mal aufrufen, werden die Rahmen verändert. Wenn man nicht speichert, bleiben die ursprünglichen Einstellungen natürlich erhalten.
